src/openrct2/ride/coaster/meta/ReverseFreefallCoaster.h

```diff
@@ -29,6 +29,8 @@ constexpr const RideTypeDescriptor ReverseFreefallCoasterRTD =
     SET_FIELD(AvailableBreakdowns, (1 << BREAKDOWN_SAFETY_CUT_OUT) | (1 << BREAKDOWN_RESTRAINTS_STUCK_CLOSED) | (1 << BREAKDOWN_RESTRAINTS_STUCK_OPEN) | (1 << BREAKDOWN_VEHICLE_MALFUNCTION)),
     SET_FIELD(LiftData, { SoundId::Null, 5, 5 }),
     SET_FIELD(UpkeepCosts, { 80, 20, 0, 0, 0, 10 }),
+    SET_FIELD(DefaultPrices, { 20, 20 }),
```

Comment:
This used to be
```    {   20, 0,  MUSIC_STYLE_ROCK,              },  // RIDE_TYPE_REVERSE_FREEFALL_COASTER```

Reply:
That was deliberate - the on-ride photo price has been set to £2.00 instead of £0.00. (Not that it currently matters, since it doesn't have one... yet.)